chaoskube/chaoskube.go

```diff
@@ -123,8 +123,10 @@ func New(client kubernetes.Interface, labels, annotations, namespaces, namespace
 
 // Run continuously picks and terminates a victim pod at a given interval
 // described by channel next. It returns when the given context is canceled.
-func (c *Chaoskube) Run(ctx context.Context, next <-chan time.Time) {
+func (c *Chaoskube) Run(ctx context.Context, maxJitter time.Duration, next <-chan time.Time) {
 	for {
+		jitter := util.RandomJitter(maxJitter)
+		time.Sleep(jitter)
```

Comment:
If we go with two sources of delay (next channel and jitter duration, see above comment) we shouldn't use sleep here because it blocks handling Ctrl+c etc.

Instead we can do another select:
```go
select {
case <-time.After(jitter):
case <-ctx.Done():
  return
}
```
after the `select` that's already below.

Reply:
I just noticed that this jitter will only increase the interval, never reduce it, right?

So, an interval of `1m` and jitter of `10s` will result in intervals between [1m, 1m10s]? (As opposed to [55m,1m5s])

Reply:
That's correct, I'm going to look at the jitterbug to see if that would fix most of the issues found here

Reply:
@linki so i think going with the jitterbug's normal distribution is what would fit well here, using a mean of 0 and standard deviation provided by the user, thoughts?

Reply:
@desponda Sgtm. Thanks for looking into it!